web/src/app.tsx

```diff
@@ -47,40 +53,112 @@ export const pages = [
   }
 ];
 
-export class App extends React.Component<{}, AppState> {
-  state: AppState = {
-    activeItem: '',
-    isNavOpen: true,
-    isDarkTheme: false
+export const App: React.FunctionComponent = () => {
+  const [isSidebarOpen, setIsSidebarOpen] = React.useState(true);
+  const [pageIndex, setPageIndex] = React.useState(0);
+  const [isDark, setDark] = React.useState(false);
+
+  const onSidebarToggle = () => {
+    setIsSidebarOpen(!isSidebarOpen);
   };
 
-  private onNavSelect = (selectedItem: { itemId: number | string }) => {
-    console.debug('onNavSelect', selectedItem);
-    this.setState({ activeItem: selectedItem.itemId });
+  const onNavSelect = (_event: React.FormEvent<HTMLInputElement>, result: { itemId: number | string }) => {
+    console.debug('onNavSelect', result);
+    setPageIndex(result.itemId as number);
   };
 
-  private onThemeSelect = (isDarkTheme: boolean) => {
+  const onThemeSelect = (isDarkTheme: boolean) => {
     console.debug('onThemeSelect', isDarkTheme);
-    this.setState({ isDarkTheme });
+    setDark(isDarkTheme);
     const htmlElement = document.getElementsByTagName('html')[0];
     if (htmlElement) {
       if (isDarkTheme) {
-        htmlElement.classList.add('pf-theme-dark');
+        htmlElement.classList.add('pf-v5-theme-dark');
       } else {
-        htmlElement.classList.remove('pf-theme-dark');
+        htmlElement.classList.remove('pf-v5-theme-dark');
       }
     }
   };
 
-  private getPageContent = (id: string) => {
-    console.debug('getPageContent', id);
+  const headerToolbar = (
+    <Toolbar id="vertical-toolbar">
+      <ToolbarContent>
+        <ToolbarItem>Netobserv</ToolbarItem>
+        <ToolbarItem align={{ default: 'alignRight' }}>
+          <ToggleGroup>
+            <ToggleGroupItem
+              icon={
+                <svg className="pf-v5-svg" viewBox="0 0 512 512">
```

Comment:
Is it netobserv icon? Where is it displayed, I don't see it anywhere

Reply:
No this is the Light / Dark theme switch. The icons are not available in PF Icons.

I can simply show text instead as:
```
          <ToggleGroup>
            <ToggleGroupItem
              text="Light"
              isSelected={!isDark}
              onClick={() => onThemeSelect(false)}
            />
            <ToggleGroupItem
              text="Dark"
              isSelected={isDark}
              onClick={() => onThemeSelect(true)}
            />
          </ToggleGroup>
```